forge-game/src/main/java/forge/game/card/CardProperty.java

```diff
@@ -1020,11 +1019,6 @@ public static boolean cardHasProperty(Card card, String property, Player sourceC
             if (!card.hasKeyword(property.substring(10))) {
```

Comment:
@tool4ever wouldn't `hasKeyword(string)` be wrong for Flashback checks? because currently Card.hasKeyword(string) checks with Real String, not with StartOf like below with "with" and "without"